docker/resource-provider-bacalhau/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Using Ubuntu as the base image
+FROM ubuntu:22.04
+WORKDIR /usr/src/app
+
+ARG WEB3_PRIVATE_KEY
```

Comment:
Could we configure this with an environment parameter passed to the container at runtime? That way we can avoid hardcoding the private key into the image.